src/utils.c

```diff
@@ -99,11 +99,11 @@ SEXP allNAR(SEXP x) {
  *   existing columns for character
  *   optionally check for no duplicates
  */
-SEXP colnamesInt(SEXP x, SEXP cols, SEXP check_dups) {
+SEXP colnamesInt(SEXP x, SEXP cols, SEXP check_dups, SEXP source) {
   if (!isNewList(x))
-    error(_("'x' argument must be data.table compatible"));
+    error(_("In %s 'x' argument must be data.table compatible"),CHAR(STRING_ELT(source, 0)));
```

Comment:
I'm not a big fan of the "In %s" but would tend to something like
`error(_("%s\n'x' argument must be data.table compatible"),CHAR(STRING_ELT(source, 0)));`
or
`error(_("'x' argument must be data.table compatible for %s"),CHAR(STRING_ELT(source, 0)));`

Ofc the source message then need to follow this style

Reply:
Done 👍 ,
Now error messages looks something like :-
```
While matching column names from 'cols' to columns in 'x'
'x' argument must be data.table compatible
```